src/scene/text-html/HTMLText.ts

```diff
@@ -1,5 +1,5 @@
 import { AbstractText, ensureTextOptions } from '../text/AbstractText';
-import { HTMLTextStyle } from './HtmlTextStyle';
+import { HTMLTextStyle } from './HTMLTextStyle';
```

Comment:
@Zyie we should add [forceConsistentCasingInFileNames](https://www.typescriptlang.org/tsconfig/#forceConsistentCasingInFileNames) to our tsconfig.json to make sure that this is caught with a type check.